.idea/runConfigurations/build_fishstick_stm32f103.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="build fishstick-stm32f103" type="CargoCommandRunConfiguration" factoryName="Cargo Command">
+    <option name="command" value="build" />
+    <option name="workingDirectory" value="file://$PROJECT_DIR$/software/fishstick-stm32f103" />
+    <option name="channel" value="DEFAULT" />
+    <option name="requiredFeatures" value="true" />
+    <option name="allFeatures" value="false" />
+    <option name="emulateTerminal" value="false" />
+    <option name="withSudo" value="false" />
+    <option name="buildTarget" value="REMOTE" />
+    <option name="backtrace" value="SHORT" />
+    <envs />
+    <option name="isRedirectInput" value="false" />
+    <option name="redirectInputPath" value="" />
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
do these files not get newlines automatically?

Reply:
No, and it's pretty annoying. They're auto-generated without newlines and if you manually add them the IDE will remove them again the next time the configuration changes

Reply:
even the newline on save setting in the IDE doesn't put it?

Reply:
They actually fight. If I manually make changes to the file (as text), it adds a new line. Then if I change the configuration through the UI, it removes the new line

Reply:
okay well this is resolved then and we will forget about it lol